src/authorizations/components/TokensTab.tsx

```diff
@@ -295,6 +296,9 @@ class TokensTab extends PureComponent<Props, State> {
     this.props.bulkDeleteAuthorizations(
       this.state.tokensSelectedForBatchOperation.map(token => token.id)
     )
+
+    const numberOfSelectedTokens = this.state.tokensSelectedForBatchOperation
+    event(`bulkAction.deleted.${numberOfSelectedTokens}.tokens`)
```

Comment:
I think we should consider changing this to similar how it is below, where instead of making the count a unique name, the event is generic and the count is a tag or a value.

```ts
event('bulkAction.tokens.deleted', {}, {count: numberOfSelectedTokens})
```

This keeps the action generic, but allows us to get a count on a specific action, rather than grouping bulk actions by how many tokens were deleted

so when we look at the stats, instead of
```
bulk action deleted 5 tokens   |   5
bulk action deleted 2 tokens   |   35
bulk action deleted 55 tokens  |   1
bulk action deleted 3 tokens   |   5
```
we have some more like:
```
bulk action tokens deleted     |   46
```

Reply:
✅